### PR TITLE
Feat/1988 areasearch preview changes

### PR DESCRIPTION
**Pull request by henrinie-nc**
image

### CODE REVIEW

File: src/application/components/_applicationPreviewSubsection.scss [hunk 1]
```diff
@@ -3,3 +3,21 @@
     margin: $spacing-l 0;
   }
 }
+
+// Selects the divs that are direct children of the root ApplicationPreviewSubsection,
+// excludes child ApplicationPreviewSubsections up to 4 levels deep
+.ApplicationPreviewSubsection:not(
```
Comment (ghost) [inline src/application/components/_applicationPreviewSubsection.scss]: Is there any other easier way to define this? Isn't the purpose to add margin bottom for the last div or something? There is `:last-child` pseudo-element.

File: src/i18n/en/common.json [hunk 2]
```diff
@@ -113,7 +113,8 @@
       "loginButton": "Log in",
       "preview": {
         "heading": "Area search application preview",
```
Comment (ghost) [inline src/i18n/en/common.json]: Title change has been done to Finnish. Should we remove translation of the heading or at least translate it ourself?

File: src/application/components/_applicationPreviewSubsection.scss (hunk 1 above)
Comment (henrinie-nc) [reply src/application/components/_applicationPreviewSubsection.scss]: The purpose is to add the margin-bottom to .ApplicationPreviewSubsection classes that are in the "top level". Your approach might work or not, I don't remember what I tried. The issue I had is that these subsections render more subsections that render more subsections, etc.
And I want to apply the style to the all "top level" classes only.

Do you have any suggestions? I don't know of a simple solution, which is why it looks like this here :smile:

File: src/i18n/en/common.json (hunk 2 above)
Comment (henrinie-nc) [reply src/i18n/en/common.json]: Good point, it should be in the English translations too. I will add it.

File: src/application/components/_applicationPreviewSubsection.scss (hunk 1 above)
Comment (henrinie-nc) [reply src/application/components/_applicationPreviewSubsection.scss]: With the updated form, it doesn't even seem to be working correctly. So how to design this for all possible recursive cases, and forms that are present or hidden, but might have answers or not? :thinking: 
image

File: src/i18n/en/common.json (hunk 2 above)
Comment (henrinie-nc) [reply src/i18n/en/common.json]: Realized that the ordering of numbers was directly taken from abstract, but it has one extra step as 1, so it was off by one. Fixed it to 3.

**Review by ghost — APPROVED** [review]

**Pull request merged**